pkgs/servers/mail/mailpit/update.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p nix coreutils curl jq nix-prefetch-git prefetch-npm-deps
+
+set -x -euo pipefail
+
+OWNER=axllent
+REPO=mailpit
+
+TARGET_TAG="$(curl https://api.github.com/repos/$OWNER/$REPO/releases/latest | jq -r '.tag_name')"
```

Comment:
This should optionally take a GITHUB_TOKEN

```suggestion
TARGET_TAG="$(curl -L -s ${GITHUB_TOKEN:+-u ":${GITHUB_TOKEN}"} https://api.github.com/repos/axllent/mailpit/releases/latest | jq -r '.tag_name')"
```